frontend/js/components/HistoryTable/HistoryTable.component.js

```diff
@@ -0,0 +1,70 @@
+// import '../../components/HistoryTable/HistoryTableRow.js'
+import Actions from '../../services/Action.service.js';
+
+class HistoryTable extends HTMLElement {
+
+    
+     constructor() {
+        super();
+        this.sessionsData = Actions.getAllSessions();
+    }
+
+    connectedCallback() {
+        this.render();
+        for(let i=0;i<this.sessionsData.length;i++)
+        {
```

Comment:
paranthesis to be on the same line as the check

Reply:
done.